tap-schema/build

```
@@ -1,6 +1,21 @@
 #!/bin/bash -ex
 ENVIRONMENT="$1"
-GIT_TAG=`echo $GITHUB_REF | sed -E 's,refs/(heads|tags)/,,' | sed -E 's,/,-,g'`
+
+echo "GITHUB_REF: $GITHUB_REF"
+if [[ "$GITHUB_REF" == refs/tags/* ]]; then
+  # Tag
+  GIT_TAG=`echo $GITHUB_REF | sed -E 's,refs/tags/,,'`
```

```suggestion
  GIT_TAG=$(echo $GITHUB_REF | sed -E 's,refs/tags/,,')
```
I think we prefer `$()` over backticks in modern shell usage. Feel free to ignore me.

---

Okay, will change this.